distribution/ddf-common/src/main/resources/etc/startup.properties

```diff
@@ -28,11 +28,11 @@
 #
 # Startup core services like logging
 #
-org/ops4j/pax/url/pax-url-aether/2.3.0/pax-url-aether-2.3.0.jar=5
```

Comment:
Do we still need to have this file? Looks like it's a full copy/paste from the karaf distribution. 

If so we should probably just take it from the karaf distribution instead of overwriting it on build.